bundlepackages/azure-dotnet.csv

```diff
@@ -195,6 +195,6 @@ azuredataappconfig,[tfm=netstandard2.0]Azure.Data.AppConfiguration
 azuremessagingeventhubsprocessor,[tfm=netstandard2.0]Azure.Messaging.EventHubs.Processor
 azuremessagingeventhubs,[tfm=netstandard2.0]Azure.Messaging.EventHubs
 azureextaspnetcoreconfigsecrt,[tfm=netstandard2.0]Azure.Extensions.AspNetCore.Configuration.Secrets
-azureextaspnetcoreconfigblobs,[tfm=netstandard2.0]Azure.Extensions.AspNetCore.DataProtection.Blobs
```

Comment:
cc @scbedd  is this expected?

Reply:
Yeah. It's harmless.

In the .NET CI, when you've got a `bundled` CSV, the only thing that's necessary is that the first column is a unique ID. The reason this is changing is because the automation uses the name of the package without separators + tolowered. What was there before was just manually entered. The actual "key" of the row is the Package Id.

Reply:
Thank you!